e2e/provision/playbooks/roles/kubeadm/tasks/main.yml

```diff
@@ -149,12 +149,53 @@
   ignore_errors: true
   changed_when: false
 
+- name: Get control plane endpoint
+  become: true
+  ansible.builtin.shell:
+    cmd: |
+      $(hostname -i | xargs -n1 | grep ^{{ ctrlplane_ep_ip_filter }})
+    executable: /bin/bash
+  register: control_plane_endpoint
+  changed_when: my_output.rc != 0
+  ignore_errors: true
+
+- name: Read kubeadm config
+  ansible.builtin.set_fact:
+    kubeadm_config_data: "{{ lookup('file', kubeadm_config) | from_yaml }}"
```

Comment:
Unbound variable on kubeadm_config

Reply:
Ah thanks, will take a look at this later today when I have some time